docs/device-data/data-types/cbg.md

```diff
@@ -27,13 +27,32 @@ The device time field is only optional for *this* data type. This is because Tid
 
 ---
 
+## Sample Interval (`sampleInterval`)
+
+With the latest generation of continuous glucose monitors, different devices may support different intervals between samples. For example, every 5 minutes or every 15 minutes between samples. In fact, some newer devices may even report multiple sample intervals from different data streams. For example, a device that reports data from both 1-minute samples and 5-minutes samples. 
+
+In order to distinguish between sample intervals and to assist with accurate statisitical calculations, the `sampleInterval` must be specified. The value must be an integer representing number of milliseconds between samples for the data stream that this datum belongs to. For example, if a device reports both 1-minute and 5-minute sample data, then the upload would contain some `cbg` data with a sample interval of 60000 (milliseconds in 1-minute) and others with a sample interval of 300000 (milliseconds in 5-minutes). The typical sample intervals are 60000 (milliseconds in 1 minute), 300000 (milliseconds in 5 minutes), and 1500000 (milliseconds in 15 minutes).
```

Comment:
There are 900,000 milliseconds in 15 minutes :wink:

Reply:
Wait, is that NEW math?

Reply:
Updated PR accordingly.